### YamlMime:ManagedReference
items:
- uid: System.Security.Authentication.InvalidCredentialException
  id: InvalidCredentialException
  children:
  - System.Security.Authentication.InvalidCredentialException.#ctor
  - System.Security.Authentication.InvalidCredentialException.#ctor(System.String)
  - System.Security.Authentication.InvalidCredentialException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Security.Authentication.InvalidCredentialException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: InvalidCredentialException
  nameWithType: InvalidCredentialException
  fullName: System.Security.Authentication.InvalidCredentialException
  type: Class
  summary: "A kivétel fordul elő, amikor hitelesítés sikertelen lesz, egy hitelesítéshez adatfolyamként, és nem indítható újra."
  remarks: "A <xref:System.Net.Security.NegotiateStream>és <xref:System.Net.Security.SslStream>osztályok throw ezt a kivételt, ha a hitelesítés nem sikerül.</xref:System.Net.Security.SslStream> </xref:System.Net.Security.NegotiateStream> Ez a kivétel azt jelzi, hogy az alapul szolgáló folyam nincs érvényes állapotban a használatával nem lehet újrapróbálkozni a <xref:System.Net.Security.NegotiateStream>vagy <xref:System.Net.Security.SslStream>példány.</xref:System.Net.Security.SslStream> </xref:System.Net.Security.NegotiateStream> Ha a hitelesítés, megpróbálkozhat azzal egy <xref:System.Security.Authentication.AuthenticationException>helyett a InvalidCredentialException történt.</xref:System.Security.Authentication.AuthenticationException>"
  syntax:
    content: 'public class InvalidCredentialException : System.Security.Authentication.AuthenticationException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Security.Authentication.AuthenticationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.Authentication.InvalidCredentialException.#ctor
  id: '#ctor'
  parent: System.Security.Authentication.InvalidCredentialException
  langs:
  - csharp
  name: InvalidCredentialException()
  nameWithType: InvalidCredentialException.InvalidCredentialException()
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException()
  type: Constructor
  assemblies:
  - System
  namespace: System.Security.Authentication
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Security.Authentication.InvalidCredentialException&quot;> </xref> osztály nem üzenettel."
  remarks: "Ez a konstruktor nincs semmi hatása."
  syntax:
    content: public InvalidCredentialException ();
    parameters: []
  overload: System.Security.Authentication.InvalidCredentialException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.Authentication.InvalidCredentialException
  langs:
  - csharp
  name: InvalidCredentialException(String)
  nameWithType: InvalidCredentialException.InvalidCredentialException(String)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(String)
  type: Constructor
  assemblies:
  - System
  namespace: System.Security.Authentication
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Security.Authentication.InvalidCredentialException&quot;> </xref> a megadott üzenet osztályra."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonságot, amelyben a szöveget a `message` paraméter.</xref:System.Exception.Message%2A> A <xref:System.Exception.InnerException%2A>tulajdonsága `null`.</xref:System.Exception.InnerException%2A>"
  syntax:
    content: public InvalidCredentialException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A <xref:System.String>, amely leírja, hogy a hitelesítés sikertelen.</xref:System.String>"
  overload: System.Security.Authentication.InvalidCredentialException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Security.Authentication.InvalidCredentialException
  langs:
  - csharp
  name: InvalidCredentialException(SerializationInfo,StreamingContext)
  nameWithType: InvalidCredentialException.InvalidCredentialException(SerializationInfo,StreamingContext)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System
  namespace: System.Security.Authentication
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Security.Authentication.InvalidCredentialException&quot;> </xref> osztály adott példányait a <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> és <xref:System.Runtime.Serialization.StreamingContext>osztályok.</xref:System.Runtime.Serialization.StreamingContext>"
  syntax:
    content: protected InvalidCredentialException (System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext);
    parameters:
    - id: serializationInfo
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> az új deszerializálásához szükséges adatokat tartalmazó példány <xref href=&quot;System.Security.Authentication.InvalidCredentialException&quot;> </xref> példány."
    - id: streamingContext
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>példány.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Security.Authentication.InvalidCredentialException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.Authentication.InvalidCredentialException
  langs:
  - csharp
  name: InvalidCredentialException(String,Exception)
  nameWithType: InvalidCredentialException.InvalidCredentialException(String,Exception)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(String,Exception)
  type: Constructor
  assemblies:
  - System
  namespace: System.Security.Authentication
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Security.Authentication.InvalidCredentialException&quot;> </xref> osztály a megadott üzenet és a belső kivételben találhatók."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonságot, amelyben a szöveget a `message` paraméter és inicializálása a <xref:System.Exception.InnerException%2A>tulajdonság a `innerException` paraméter értéke.</xref:System.Exception.InnerException%2A> </xref:System.Exception.Message%2A>"
  syntax:
    content: public InvalidCredentialException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A <xref:System.String>, amely leírja, hogy a hitelesítés sikertelen.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "A <xref:System.Exception>, amely a jelenlegi kivétel okát.</xref:System.Exception>"
  overload: System.Security.Authentication.InvalidCredentialException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Security.Authentication.AuthenticationException
  isExternal: false
  name: System.Security.Authentication.AuthenticationException
- uid: System.Security.Authentication.InvalidCredentialException.#ctor
  parent: System.Security.Authentication.InvalidCredentialException
  isExternal: false
  name: InvalidCredentialException()
  nameWithType: InvalidCredentialException.InvalidCredentialException()
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException()
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.String)
  parent: System.Security.Authentication.InvalidCredentialException
  isExternal: false
  name: InvalidCredentialException(String)
  nameWithType: InvalidCredentialException.InvalidCredentialException(String)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Security.Authentication.InvalidCredentialException
  isExternal: false
  name: InvalidCredentialException(SerializationInfo,StreamingContext)
  nameWithType: InvalidCredentialException.InvalidCredentialException(SerializationInfo,StreamingContext)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Security.Authentication.InvalidCredentialException.#ctor(System.String,System.Exception)
  parent: System.Security.Authentication.InvalidCredentialException
  isExternal: false
  name: InvalidCredentialException(String,Exception)
  nameWithType: InvalidCredentialException.InvalidCredentialException(String,Exception)
  fullName: System.Security.Authentication.InvalidCredentialException.InvalidCredentialException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.Authentication.InvalidCredentialException.#ctor*
  parent: System.Security.Authentication.InvalidCredentialException
  isExternal: false
  name: InvalidCredentialException
  nameWithType: InvalidCredentialException.InvalidCredentialException
